tests/tensorflow/targets.jsonnet

```diff
@@ -13,12 +13,9 @@
 // limitations under the License.
 
 local nightly = import 'nightly/targets.jsonnet';
-local r2_11 = import 'r2.11/targets.jsonnet';
 local r2_12 = import 'r2.12/targets.jsonnet';
 
 // Add new versions here
 std.flattenArrays([
   nightly,
-  r2_11,
-  r2_12,
```

Comment:
nit: if you remove `r2_12` here, then you should remove the whole `r2.12` folder since they won't be in the build. Otherwise, setting `schedule: null` and making sure there are no overrides should be sufficient

Reply:
Done. Thanks!